### PR TITLE
Started Working on Adopt a Dog Website

### PR DESCRIPTION
**Pull request by Hamediaj**
Started working on html for the website that includes:

- Homepage
- Dog Page
- Blog Page

### CODE REVIEW

File: blog.html [hunk 1]
```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html lang="zxx" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Adopt A Dog</title>
+  </head>
+  <body>
+    <header>
+      <figure class="company-logo">
```
Comment (jimmynono) [inline blog.html]: Great use of figure and I like the way you used kebab casing on the class.

File: blog.html [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+        <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</p>
+
+        <p>Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+
+      </article>
+
+      <article class="blog">
+
+        <figure>
+          <img src="images/blog-2.jpg" alt="Dog walker walking four dogs">
```
Comment (jimmynono) [inline blog.html]: Great job on using descriptive alt tags!

File: blog.html [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+      </article>
+
+
+    </main>
+
+    <footer>
+      <figure>
+        <img src="images/adoptadog-logo.svg" alt="Company Logo">
+      </figure>
+      <small>Corrum volorit iandae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis re ea erum, qui doluptae</small>
```
Comment (jimmynono) [inline blog.html]: I'm not a huge fan of using the small element. It's fine to use, but I'd rather you wrap it in a span (or div) an style from there. Relying on the browser to render small right can be somewhat tricky. This is just my opinion and what you're doing isn't wrong, but I'd steer you towards a span

File: checkout.html [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+      </nav>
+
+    </header>
+    <main>
+
+      <h1>Checkout</h1>
+
+      <form class="checkout" action="checkout.html" method="get">
+
+        <label class="name">Name
```
Comment (jimmynono) [inline checkout.html:52]: make sure to give each label and input the for attribute on the label and the id on the input

File: checkout.html [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+        <label class="address">Street Address
+          <input type="text" name="applicantAddress">
+        </label>
+
+        <label class="city">City
+          <input type="text" name="applicantCity">
+        </label>
+
+        <label class="state">State
+          <select id="state" name="applicantState"></select>
```
Comment (jimmynono) [inline checkout.html:69]: don't forget to put in a few options, just one or two states will be fine

File: checkout.html [hunk 6]
```diff
@@ -0,0 +1,127 @@
...
+        </label>
+
+        <h3>Dogs Adopted</h3>
+
+        <fieldset class="adopted-lucas">
+          <figure>
+            <img src="images/lucas-checkout.jpg" alt="Picture of Lucas">
+          </figure>
+          <h3>Lucas</h3>
+          <p><strong>Cost to Adopt:</strong> $123.45</p>
```
Comment (jimmynono) [inline checkout.html:100]: strong here is like the small in the footer. I'd steer you towards using a span and style it in css

File: checkout.html [hunk 7]
```diff
@@ -0,0 +1,127 @@
...
+        </fieldset>
+
+        <fieldset class="adopted-duffy">
```
Comment (jimmynono) [inline checkout.html:103]: for both of these dog sections you'll want to use the same class name for each. That way you can use one class on two elements and style the same.

File: dogs.html [hunk 8]
```diff
@@ -0,0 +1,171 @@
...
+          <li class="navlist"><a href="blog.html">Blog</a></li>
+        </ul>
+      </nav>
+
+    </header>
+    <main>
+
+        <h1>Here are all of our dogs!</h1>
+
+      <section class="dog-photos">
```
Comment (jimmynono) [inline dogs.html]: You're on the right path for these tiles. You will want to have some sort of wrapping element around all of the dog tiles to lay them out.

File: dogs.html [hunk 9]
```diff
@@ -0,0 +1,176 @@
...
+
+    </header>
+    <main>
+
+        <h1>Here are all of our dogs!</h1>
+    <div class="dog-page">
+
+      <section class="dog-photos">
+        <figure>
+          <img src="images/murphy-card.jpg" alt="Picture of Murphy" width="60%">
```
Comment (tylervanbrocklin) [inline dogs.html]: You'll want to include padding in your dog-photos css so the content is padded inside of each tile.

File: dogs.html [hunk 10]
```diff
@@ -0,0 +1,176 @@
...
+        <p><strong>Cost to Adopt:</strong> $123.45</p>
+        <p>Corrum volorit iandae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis re ea erum, qui doluptae</p>
+        <button type="button" name="button">Adopt</button>
+      </section>
+
+      </div>
+
+    </main>
+
+    <footer>
```
Comment (tylervanbrocklin) [inline dogs.html:174]: Would also recommend adding padding between your dog tiles and footer.

File: blog.html [hunk 11]
```diff
@@ -0,0 +1,101 @@
...
+            <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</p>
+
+            <p>Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+
+          </article>
+      </div>
+
+      <div class="blog-page">
+
+          <figure class="blog-image">
```
Comment (tylervanbrocklin) [inline blog.html]: Would suggest you use flexbox for your blog-page css. So when the screen size shrinks, your text will go below the image. Right now it just gets squished together

File: index.html [hunk 12]
```diff
@@ -0,0 +1,153 @@
...
+  <body>
+    <header>
+
+      <figure class="company-logo">
+        <img src="images/adoptadog-logo.svg" alt="Company Logo">
+      </figure>
+
+      <figure class="cart-icon">
+        <img src="images/cart-icon.svg" alt="Cart Icon">
+      </figure>
```
Comment (tylervanbrocklin) [inline index.html]: Make sure to add the checkout button so there is a way to navigate to your checkout page

File: blog.html [hunk 13]
```diff
@@ -5,6 +5,12 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <link rel="stylesheet" href="./styles/stylesheet.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
```
Comment (jimmynono) [inline blog.html:7]: To make your life easier I'd recommend only using two fonts. It's one more thing to complicate

File: dogs.html (hunk 9 above)
Comment (jimmynono) [reply dogs.html]: also, don't use styling in the HTML. Pull all of that out and use CSS to style elements

File: styles/stylesheet.css [hunk 14]
```diff
@@ -0,0 +1,167 @@
...
+
+.intro-article{
+  display:flex;
+  flex-direction:column;
+  padding-left: 3rem;
+}
+
+.main-dogs {
+  display: grid;
+  grid-template-columns: 25% 25% 25% 25%;
```
Comment (jimmynono) [inline styles/stylesheet.css]: Great! We'll show you later on how to make that different for different widths.

File: index.html [hunk 15]
```diff
@@ -0,0 +1,171 @@
...
+        </section>
+
+        <section class="intro-dogs">
+          <figure class="intro-img">
+            <img class="dogimage" src="images/violet-card.jpg" alt="Picture of Violet" onclick="dogInformation('Violet', 'Labrador', '$123.45')">
+          </figure>
+          <h3>Violet</h3>
+          <p><span class="dogs">Cost to Adopt:</span> $123.45</p>
+          <p>Corrum volorit iandae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis re ea erum, qui doluptae</p>
+          <button  class="dogs-button" type="submit" onclick="adoptionFee(123.45)">Adopt</button>
```
Comment (tylervanbrocklin) [inline index.html:124]: You do not need to include `type="submit"` this is just needed for buttons in forms

File: index.html [hunk 16]
```diff
@@ -0,0 +1,211 @@
...
+          <h3>Murphy</h3>
+          <p><span class="dogs">Cost to Adopt:</span> $123.45</p>
+          <p>Corrum volorit iandae nimaxim cum restia volor reicid ut et etur sunt arum rendae</p>
+          <!-- <button class="dogs-button" type="button" name="button">Adopt</button> -->
+
+          <button  class="dogs-button" type="submit" onclick="adoptionFee(123.45)">Adopt</button>
+
+        </section>
+
+        <section class="intro-dogs">
```
Comment (tylervanbrocklin) [inline index.html:77]: Would see if you could also add the dog tile animations to this page as well!

File: checkout.html [hunk 17]
```diff
@@ -0,0 +1,147 @@
...
+          <input type="text" name="applicantAddress">
+        </label>
+
+        <label class="city">City
+          <input type="text" name="applicantCity">
+        </label>
+
+        <label class="state">State
+          <select id="state" name="applicantState"></select>
+        </label>
```
Comment (tylervanbrocklin) [inline checkout.html:70]: Like James has mentioned, make sure to put a few options for state

File: checkout.html [hunk 18]
```diff
@@ -0,0 +1,147 @@
...
+          </label>
+
+          <label>No
+            <input type="radio" name="previouslyAdopted" value="No">
+          </label>
+
+        </fieldset>
+
+        <label class="pickup">Pickup Location
+          <select id="pickup" name="pickupLocation"></select>
```
Comment (tylervanbrocklin) [inline checkout.html:90]: Put a few options here as well

File: index.html [hunk 19]
```diff
@@ -0,0 +1,211 @@
...
+      <div class="header-figures">
+
+        <figure class="company-logo">
+          <img src="images/adoptadog-logo.svg" alt="Company Logo">
+          <figcaption class="company">Seattle’s premiere dog adoption agency</figcaption>
+        </figure>
+
+        <figure class="cart-icon">
+          <img src="images/cart-icon.svg" alt="Cart Icon">
+          <a class="total" href="checkout.html" class="checkout">Checkout</a>
```
Comment (tylervanbrocklin) [inline index.html:27]: Your cart should display a total. So as you click adopt, the cart total is reflected (using jQuery) here

**Pull request closed without merging**